js/modules/k6/grpc/client.go

```diff
@@ -194,9 +194,9 @@ func (c *Client) Invoke(
 		return nil, fmt.Errorf("method %q not found in file descriptors", method)
 	}
 
-	p, err := c.parseParams(params)
+	p, err := c.parseInvokeParams(params)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("invalid grpc.Invoke() parameters: %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("invalid grpc.invoke() parameters: %w", err)
```

https://k6.io/docs/javascript-api/k6-net-grpc/client/client-invoke